botx/models/entities.py

```diff
@@ -80,6 +80,23 @@ def generate_mention_id(cls, mention_id: Optional[UUID]) -> UUID:  # noqa: N805
         """
         return mention_id or uuid4()
 
+    @validator("mention_data", pre=True, always=True)
+    def ignore_empty_data(
+        cls, mention_data: Union[ChatMention, UserMention, Dict],  # noqa: N805
+    ) -> Optional[Union[ChatMention, UserMention, Dict]]:
+        """Pass empty dict into mention_data as None.
+
+        Arguments:
+            mention_data: dict of mention's data.
+
+        Returns:
+             Mention's data if is not empty or None.
+        """
+        if not mention_data:
```

Comment:
Nit: `if mention_data == {}` will be more accurate.

Reply:
`if mention_data == {}` it is WPS warning.

Reply:
Pls add `noqa`. If we add new type, which also will be interpreted as `False`, it will lead to bug.